examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,57 @@
+"""
+=============================
+Plot positions on a blank map
+=============================
+
+This example showcases how to plot positions on a blank map.
+It is commonly used to show HelioProjective Cartesian (HPC) positions of events that occurred at
+different times and therefore no single observation is appropriate.
```

Comment:
I think this sentence is a bit confusing, and this example could also be extended to plot other points that are not in HPC coordinate system (i.e. just define a different SkyCoord). How about:

```suggestion
It is often useful to plot coordinate positions of events on a blank helioprojective coordinate map. 
In this example, we create an empty map with a WCS defined by a helioprojective frame as observed from Earth at a certain time, and show how you can plot different coordinates on it.
```